### DISCUSSION TITLE
Error transcode HW enabled

### DISCUSSION THREAD

**Discussion by kraizelburg** [stashapp/stash#4942] [Q&A]
Hi, I just instaled last 0.26 release and when I try to transcode I am getting the followin errors:
Error   
[transcode] ffmpeg error when running command </root/.stash/ffmpeg -hide_banner -v error -hwaccel auto -ss 11.095836 -i /data/videos/video.mp4 -c:v libx264 -pix_fmt yuv420p -preset veryfast -crf 25 -sc_threshold 0 -movflags frag_keyframe+empty_moov -vf scale=-2:480 -ac 2 -f mp4 pipe:>: [AVHWDeviceContext @ 0x5c919c0] No VA display found for any default device.
Device creation failed: -22.
Device creation failed: -38.

No matter what file I try to play I always get the same error. I am using zimaboard with debian 12 server installed.

Any ideas? thanks

**Comment by feederbox826** [comment]
- you have not passed through your /dev/dri* devices to your docker container
- your iGPU does not have a displayout so hwaccel is not possible

**Comment by kraizelburg** [comment]
> * you have not passed through your /dev/dri* devices to your docker container
> * your iGPU does not have a displayout so hwaccel is not possible

Hi, I have already passed through /dev/dri/renderD128 in docker-compose file but I still see in Stash logs [InitHWSupport] Supported HW codecs [0]:

**Comment by WithoutPants** [comment]
See also: https://github.com/stashapp/stash/issues/305#issuecomment-2153775341
